apiserver/facades/controller/undertaker/mock_test.go

```diff
@@ -147,6 +149,10 @@ func (m *mockModel) Tag() names.Tag {
 	return names.NewModelTag(m.uuid)
 }
 
+func (m *mockModel) ModelTag() names.ModelTag {
```

Comment:
Given we already use Tag() - see above - let's stick with that instead of having both Tag() and ModelTag()